tools/flang2/flang2exe/llassem.cpp

```diff
@@ -1698,6 +1704,18 @@ write_comm(void)
 
     align_value = CACHE_ALIGN + 1;
 
+    /*
+      Create a tmp file and capture the initialization data.
+      Emitting the cmn blk var defn now will miss out the dbg info
+      generated for other references of this variable.
+    */
+    FILE *llvm_ir_bkup = gbl.asmfil;
+    FILE *cmn_blk_ir;
+    if ((cmn_blk_ir = tmpfile()) == NULL)
+      errinfo(F_0005_Unable_to_open_temporary_file);
```

Comment:
Shouldn't this call `errfatal` just like all the failing `fopen` attempts in `tools/flang2/flang2exe/main.cpp`?

Reply:
My bad. Changed.